### ISSUE TITLE
High contrast mode

### ISSUE THREAD

**Issue by ColinMcNeil** [SquaredLabs/squaredlabs.uconn.edu#200]
Add toggle to enable a "high contrast" mode.

**Comment by ColinMcNeil** [comment]
On branch https://github.com/SquaredLabs/squaredlabs.uconn.edu/tree/high-contrast

Should be live at http://high-contrast.squaredlabs.ci.core.uconn.edu/labs
